src/client.test.js

```diff
@@ -60,7 +62,8 @@ describe('Client side', () => {
       <BadComponent />
     </ErrorFallback>)
 
-    expect(component.html()).toBe('<div></div>')
+    expect(component.html()).toBe(null)
+    expect(component.html()).toMatchSnapshot()
```

Comment:
Since it doesn't render the extra div, we now default to null. This is _possibly_ a breaking change.